### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Ruslan-Mykhalchak**
[DEMO LINK](https://Ruslan-Mykhalchak.github.io/react_counter/)

### CODE REVIEW

**Review by Artem5457 — APPROVED** [review]
It works correctly

File: src/App.scss [hunk 1]
```diff
@@ -1 +1,42 @@
...
+  cursor: pointer;
+ 
+
+  &:hover {
+    background-color: rgb(201, 111, 0);
+    transform: scale(120%);
+    
+  }
+
+}
```
Comment (aliceblunt8) [inline src/App.scss:42]: remove empty spaces from your scss

```suggestion
body {
  background-color: bisque;
}

.container {
  width: 400px;
  margin: 10px auto;
}

.title {
  text-align: center;
  color: rgb(201, 111, 0);
}

.buttons {
  display: flex;
  justify-content: space-between;
}

.button {
  border: none;
  background-color: rgb(241, 170, 82);
  width: 80px;
  height: 30px;
  border-radius: 10px;
  transition: 0.5s;
  cursor: pointer;
 
  &:hover {
    background-color: rgb(201, 111, 0);
    transform: scale(120%);
  }
}
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,16 +1,79 @@
...
+
+  add100 = () => {
+    this.setState((prevState) => ({
+      count: prevState.count + 100,
+    }));
+  };
+
+  increase = () => {
+    this.setState((prevState) => ({
+      count: prevState.count % 5 === 0 ? (prevState.count + 101) : (prevState.count + 1),
```
Comment (aliceblunt8) [inline src/App.tsx:27]: this function should call addOne or add100 depending on `prevState.count % 5 === 0`

File: src/App.tsx [hunk 3]
```diff
@@ -1,16 +1,79 @@
...
+    }));
+  };
+
+  reset = () => {
+    this.setState((prevState) => ({
+      count: prevState.count * 0,
+    }));
+  };
```
Comment (aliceblunt8) [inline src/App.tsx:35]:
```suggestion
  reset = () => {
    this.setState({ count: 0 });
  };
```

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**